bot.py

```diff
@@ -37,14 +37,12 @@ def surrender(message):
 @bot.message_handler(content_types=['text'])
 def play_game(message):
 
+    if not game_dict:
+        bot.send_message(message.from_user.id, 'you need to start a new game first, type /newgame')
```

Comment:
There are many places where 'message.from_user.id' is used. I would suggest creating a new variable 'user_id' to simplify several lines below.